routes/demos.js

```diff
@@ -62,7 +62,7 @@ module.exports = function (express) {
             else {
               // Define number of unnecessary directories to be omitted while downloading.
               const numDirsToCut = sampleUrl.pathname.match(/TAU\/examples\/mobile|wearable\/UIComponents/g) ? 4 : 0;
-              exec(`wget --page-requisites --convert-links --no-host-directories --cut-dirs=${numDirsToCut} --directory-prefix ${projectPath} ${sampleUrl}`, (error, stdout, stderr) => {
+              exec(`wget -r --page-requisites --convert-links --no-host-directories --cut-dirs=${numDirsToCut} --directory-prefix ${projectPath} ${sampleUrl}`, (error, stdout, stderr) => {
```

Comment:
Please replace -r with  --recursive for better code readability.

Reply:
done